<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ui_ArcTangentialOptions</class>
 <widget class="QWidget" name="Ui_ArcTangentialOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>258</width>
    <height>24</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>248</width>
    <height>22</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Tangential Arc Options</string>
  </property>
  <widget class="Line" name="sep1">
   <property name="geometry">
    <rect>
     <x>124</x>
     <y>0</y>
     <width>16</width>
     <height>24</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="frameShape">
    <enum>QFrame::VLine</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="leRadius">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>0</y>
     <width>60</width>
     <height>22</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="toolTip">
    <string>Radius of the tangential arc</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leAngle">
   <property name="geometry">
    <rect>
     <x>192</x>
     <y>0</y>
     <width>60</width>
     <height>22</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="toolTip">
    <string>Total subtending angle of the tangential arc</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbRadius">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>70</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string extracomment="Draw Tangential Arc by the given radius">Radius</string>
   </property>
   <property name="iconSize">
    <size>
     <width>12</width>
     <height>12</height>
    </size>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbAngle">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>0</y>
     <width>65</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string extracomment="Draw Tangential Arc by the given radius">Angle</string>
   </property>
   <property name="iconSize">
    <size>
     <width>12</width>
     <height>12</height>
    </size>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>leRadius</tabstop>
  <tabstop>leAngle</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>leRadius</sender>
   <signal>editingFinished()</signal>
   <receiver>QG_ArcTangentialOptions</receiver>
   <slot>on_leRadius_editingFinished()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leAngle</sender>
   <signal>editingFinished()</signal>
   <receiver>QG_ArcTangentialOptions</receiver>
   <slot>on_leAngle_editingFinished()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
